src/org/opensolaris/opengrok/web/PageConfig.java

```diff
@@ -752,6 +752,20 @@ public String getRequestedProjectsAsString() {
         return requestedProjects;
     }
     
+	public SortedSet<String> getAllRequestedProjects() {
+		    TreeSet<String> set = new TreeSet<>();
+            Set<Project> allProjects = getProjectHelper().getAllProjects();
```

Comment:
so basically this whole method is useless, call to `getProjectHelper().getAllProjects()` is enough anywhere when you would call this method